sysutils/kubectx/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        ahmetb kubectx v0.6.3
+github.tarball_from archive
+
+name                kubectx
+version             0.6.3
+categories          sysutils
+platforms           darwin
+supported_archs     x86_64
+license             Apache-2
+
+maintainers         @gearnode
+
+checksums           rmd160 4933fbce2b323410ec461f86c8350b560790fc8d \
+                    sha256 4e501055d37829d3b3834c5883017a785e56bf9b8b8712040a504a7ead04472d
+
+description         Switch faster between clusters and namespaces in kubectl
+long_description    Switch faster between clusters and namespaces in kubectl
+
+use_zip             yes
+use_configure       no
+
+worksrcdir          ${name}-${version}
+
+depends_run         port:kubectl
+
+build {
+}
```

Comment:
There's really no build phase? Then how does this work?

Reply:
It looks like the files are bash scripts. So the `supported_archs` line should be changed to:

`supported_archs noarch`